README.rst

```diff
@@ -11,88 +11,88 @@ more information about the backend service.
-          `@@@  '@@@@@@@@@@@@@       @@@@@@@@@@@@@`  @@@
-         '@@@` .@@@@@@@@@@@@@@@    `@@@@@@@@@@@@@@@  @@@@`
-         @@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        '@@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        ,:::;  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  ,:::
-           :@  ,@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  #@
-           @@@  +@#+#@@@@@@@@@@@@@@@@@@@@@@@@@#+#@.  @@@
-           @@@@        '@@@@@@@@@@@@@@@@@@@.        @@@@
-           @@@  @@@@@@+  @@@@@@@@@@@@@@@@@  @@@@@@;  @@@
-           @@  @@@@@@@@@  @@@@@@@@@@@@@@@ `@@@@@@@@@  @+
-              @@@@@@@@@@@ :@@@@@@@@@@@@@  @@@@@@@@@@@ '
-             `@@@@@@@@@@@       ```      ,@@@@@@@@@@@
-             `@@@@@@   '@                :@:   @@@@@@
-              @@@@@:                           @@@@@@
-               @@@@@                           @@@@@
-                @@@@#                         @@@@'
-
-Platform requirements
-=====================
-
-Aiven Client has been tested and developed on Linux and Mac OS X systems.
-It is a Python program that works with Python 3.6 or newer versions.
-The only external dependency is Requests_ (and certifi_ on Windows/OSX).
+.. contents::
+
+
+.. _platform-requirements:
+
+Getting Started
+===============
+
+Requirements:
+
+*  Python 3.6 or later
+
+*  Requests_
+
+*  For Windows and OSX, certifi_ is also needed
 
 .. _`Requests`: http://www.python-requests.org/
 .. _`certifi`: https://certifi.io/
 
-Installation
-============
+.. _installation:
 
-From PyPI (Linux/OSX)::
+Install from PyPi
+-----------------
+
+Pypi installation is the recommended route for most users::
 
   $ python3 -m pip install aiven-client
 
-From PyPI (Windows)::
 
-  c:\> python3 -m pip install aiven-client
+Build an rpm Package
+--------------------
 
-Build an RPM package (Linux)::
+It is also possible to build an rpm::
 
   $ make rpm
 
-Basic Usage
-===========
+Check Installation
+------------------
 
-* NOTE: On Windows you may need to use ``python3 -m aiven.client`` instead of ``avn``.
-* All commands will output the raw REST API JSON response with ``--json``
+To check that the tool is installed and working, run it without arguments::
 
-Help command
-------------
+  $ avn
 
-::
+If you see usage output, you're all set.
 
-  $ avn help
+  **Note:** On Windows you may need to use ``python3 -m aiven.client`` instead of ``avn``.
 
-List all commands. You can use the help to also search for matching commands,
-for example
+Log In
+------
 
-::
+The simplest way to use Aiven CLI is to authenticate with the username and
+password you use on Aiven::
 
-  $ avn help kafka topic
+  $ avn user login <you@example.com>
 
-would list all commands that have kafka *and* topic in their description.
+The command will prompt you for your password.
+
+.. _help-command:
+.. _basic-usage:
+
+Usage
+=====
+
+Some handy hints that work with all commands:
+
+*  The ``avn help`` command shows all commands and can _search_ for a command,
```

Comment:
GitHub rendres it like this:
![image](https://user-images.githubusercontent.com/3449635/105670840-5c5eb180-5eea-11eb-9e9b-9ef448986893.png)

You probably wanted `*search*` (*search*).

Reply:
Good catch, fixed